src/main/java/com/limechain/transaction/TransactionState.java

```diff
@@ -125,4 +126,16 @@ public void removeExtrinsicFromPool(Extrinsic extrinsic) {
     public byte[] addToPool(ValidTransaction validTransaction) {
         return transactionPool.insert(validTransaction);
     }
+
+    public void addAndReturnIgnoredPeers(Extrinsic extrinsic, PeerId peerId) {
+        if (existsInQueue(extrinsic)) {
+            for (ValidTransaction validTransaction : transactionQueue) {
+                if (validTransaction.getExtrinsic().equals(extrinsic)) {
+                    validTransaction.getIgnore().add(peerId);
```

Comment:
Can you change the name of `ignore` in the `ValidTransaction` class to something a bit more informative like `peersToIgnore` or smth similar

Reply:
I created getter [getIgnoredPeers] and setter [addPeerToIgnore] for ```ignore``` field, because it is used by the runtime and renaming it results in in INALID_TRANSACTION.